src/game.cpp

```diff
@@ -10586,6 +10586,8 @@ point game::update_map( int &x, int &y )
     if( shift == point_zero ) {
         // adjust player position
         u.setpos( tripoint( x, y, get_levz() ) );
+        // Update what parts of the world map we can see
+        update_overmap_seen();
```

Comment:
Would you mind adding a comment for the sake of the future developer scratching their head over why this needs to be here?
```suggestion
        // We need this call because even if the map hasn't shifted we may have changed z-level and can now see farther
        // TODO: only make this call if we changed z-level
        update_overmap_seen();
```